### PR TITLE
Merge branch 'develop' of https://github.com/ResilienX/fraihmwork-api…

### PR DESCRIPTION
**Pull request by MS-ResilienX**

### CODE REVIEW

File: Deprecated/BlocklistingApi.yaml
Comment (MS-ResilienX) [inline Deprecated/BlocklistingApi.yaml:1]: This diff isn't very helpful. The two BlocklistingApi* files from develop were merely copied into the Deprecated folder.

File: FraihmworkHealthAndIntegrityApi.yaml [hunk 1]
```diff
@@ -25,6 +25,9 @@ tags:
   
 - name: UUID Control
   description: Functions for helping manage UUIDs for components and faults
+  
+- name: Blocklist
+  description: Blocklist management apis
```
Comment (BW-ResilienX) [inline FraihmworkHealthAndIntegrityApi.yaml:30]: To match other tag descriptions.
```suggestion
  description: Blocklist management api endpoints
```

**Review by BW-ResilienX — CHANGES_REQUESTED** [review]

File: FraihmworkHealthAndIntegrityApi.yaml [hunk 2]
```diff
@@ -1054,6 +1057,210 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Blocklist'
+        401:
+          description: Not authenticated
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
```
Comment (GD-ResilienX) [inline FraihmworkHealthAndIntegrityApi.yaml:1080]: Let's toss some 403's here in the blocklister apis, for accuracy and consistency

File: FraihmworkHealthAndIntegrityApi.yaml [hunk 3]
```diff
@@ -1054,6 +1057,210 @@ paths:
...
+        500:
+          description: Server error
+          content: 
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+      security:
+      - OAuth2: [fraihmwork.blocklist.read]
```
Comment (GD-ResilienX) [inline FraihmworkHealthAndIntegrityApi.yaml:1094]: I think fraihmwork.blocklist.manage should probably be able to read the blocklist, we moved towards this in the rest of the endpoints in this spec.

**Review by GD-ResilienX — CHANGES_REQUESTED** [review]

File: FraihmworkHealthAndIntegrityApi.yaml [hunk 4]
```diff
@@ -1393,3 +1680,5 @@ components:
             fraihmwork.fault.admin: Client may create, read, update, or delete any fault
             fraihmwork.fault.write: Client may create new faults, update and delete faults they create, and read all faults
             fraihmwork.fault.read: Client may query for any detailed fault information that FRAIHMWORK can provide
+            fraihmwork.blocklist.manage: Client can modify the blocklist
```
Comment (BW-ResilienX) [inline FraihmworkHealthAndIntegrityApi.yaml:1683]: Should we try to match the language of the component and fault scopes?

**Review by BW-ResilienX — APPROVED** [review]

**Review by GD-ResilienX — APPROVED** [review]

**Pull request merged**